styles/variables-base.less

```diff
@@ -258,6 +265,7 @@
 @agate-item-side-outset: @agate-component-spacing;
 @agate-item-height: 60px;
 @agate-item-height-large: 72px;
+@agate-item-line-height: 60px;
```

Comment:
If this is necessary, it should be a referential value like the following to be easier to work with:
```css
@agate-item-line-height: @agate-item-height;
```